roles/rclone_serve_restic/README.md

```diff
@@ -0,0 +1,21 @@
+# `famedly.base.rclone_serve_restic` ansible role for multi-tenancy append-only backups to s3 using rclone
+
+## Required Options:
+
+this role supports a single s3 bucket as a backend, you can set the required options like so
+``` yml
+rclone_serve_restic_backend_config:
+  endpoint: "top secret"
+  access_key_id: "middle secret"
+  secret_access_key: "bottom secret"
+
+  # you can just overwrite the defaults with the following
+  type: s3
+  provider: minio
+  env_auth: false
+  region: home-sweet-home
+  acl: private
+
+```
+
+you also NEED to write secrets to the htpasswd file yourself, or else it will be exposed WITHOUT AUTHENTICATION!!
```

Comment:
README should state/explain what/where "the htpasswd file" is, if it is available as a variable etc...